docs/send-data/opentelemetry-collector/time-reference.md

```diff
@@ -0,0 +1,71 @@
+---
+id: time-reference
+title: OTel Timestamps Date Formats
+description: Learn how Sumo Logic manages OTel timestamps and the configuration options that are available.
+keywords:
+  - time-stamp
+  - otel-collector
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We support several options for timestamps. When collecting log data, the timestamp attached to messages is vital, both for the integrity of the data in your account, and for accurate query results.
```

Comment:
I'd clarify that the doc is about how they need to format timestamps in their OTel Collector configurations